pyaedt/edb_core/EDB_Data.py

```diff
@@ -3121,6 +3121,15 @@ def partname(self):
         """
         return self.edbcomponent.GetComponentDef().GetName()
 
+    @partname.setter
+    def partname(self, name):
+        """Set component Part Name.
+
+        Returns
+        -------
+        """
+        return self.edbcomponent.GetComponentDef().SetName(name)
```

Comment:
@ring630 I am just double checking. Do you expect to return a value in the setter?
```suggestion
        self.edbcomponent.GetComponentDef().SetName(name)
```